js/fetch/addProduct.js

```diff
@@ -0,0 +1,16 @@
+export const addProduct = async (data) => {
```

Comment:
Consider having these api call functions in a single file as they are tightly related anyways. 😃